docs/usage/syncing-instances.md

```diff
@@ -2,11 +2,60 @@
 sidebar_position: 4
 ---
 # Syncing Instances
-So, how to sync every instance of HuronOS at the same time?, you might ask. The answer is: using a directives file. Where to retrieve this file from is defined when a HuronOS usb is created, and that file is retrieved every minute for every HuronOS instance (although that might not happen at the same time) so, if the directives file url is correctly configured and set on the usbs, every instance should be in sync with the directives file wherever that file is stored.
 
+**Remote syncing** is one of the flagship features of huronOS, allowing you to synchronize preferences, configurations, and behavior directives with **several huronOS instances** (running installations of huronOS) using something we call *the directives file*. This file is a plain text file with a specific syntax known as *huronOS Directives File Syntax*, resulting in a *.hdf* file.
 
-# Troubleshoot
-### My directives do not sync
-- Be sure your file is accessible from other machines.
-- Verify the instance you're trying to sync has an internet connection available.
-- Verify that your expected directives file reflects the one stored at your server.
\ No newline at end of file
+To sync an instance, you need to specify a directives file URL that can be accessed via HTTP or HTTPS. This URL can be a domain name or an IP address. The huronOS **sync manager** polls this file every minute, comparing it to the local version. If any changes are detected, the sync manager processes them and triggers the necessary updates to align the system with the new rules. Be aware you can expect up to 2 minutes from the edition to the full propation.
```

Comment:
I think you meant "the full propagation"? Not sure about the term, just to make sure with you :)